deeprankcore/trainer.py

```diff
@@ -548,12 +619,12 @@ def _eval( # pylint: disable=too-many-locals
         for _, data_batch in enumerate(loader):
```

Comment:
```suggestion
        for data_batch in loader:
```

Reply:
Changed this in commit 9ad784c964ef77876f71e39599c150dcace6bb22.

Reply:
Changed in 7b6b47bf79ccf03fa1f9a0479f054316a0d9bff4